_pages/home-next.md

```diff
@@ -1,5 +0,0 @@
----
```

Comment:
## Delete: `home-next.md`
I think we don't need the extra step of having another file for `home.md`. I'm guessing this was set up here as a way to make and share/discuss tentative edits to the site without updating the site itself, and we'd need one of these files for each page.

 We can do this instead using (a) a pull request, (b) discussing/writing tentative content elsewhere, like in a google doc. If we accidentally push a tentative/bad edit, we can always roll back to a previous commit.